app/timetables/models.py

```diff
@@ -118,3 +118,25 @@ class Dish(TimestampMixin):
 
     def __str__(self):
         return self.name
+
+
+class MenuItem(TimestampMixin):
+    """
+    Model representing a Menu Item.
+
+    A MenuItem represents the particular meal combination option that is to be
+    served on a given cycle-day of a particular timetable.
+    """
+
+    timetable = models.ForeignKey(Timetable)
+    cycle_day = models.PositiveSmallIntegerField(
+        validators=[MinValueValidator(1)]
+    )
+    meal = models.ForeignKey(Meal)
+    meal_option = models.ForeignKey(MealOption)
+
+    class Meta:
+        unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+    def __str__(self):
+        return '%s %s' % (self.cycle_day, self.meal)
```

Comment:
Can we use the newer and more advanced string formatting in Python `.format()`? Per this [StackOverflow discussion](http://stackoverflow.com/questions/5082452/python-string-formatting-vs-format)